extensions/npm/src/tasks.ts

```diff
@@ -187,14 +196,12 @@ async function detectNpmScripts(context: ExtensionContext, showWarning: boolean)
 				const paths = await workspace.findFiles(relativePattern, '**/{node_modules,.vscode-test}/**');
 				for (const path of paths) {
 					if (!isExcluded(folder, path) && !visitedPackageJsonFiles.has(path.fsPath)) {
-						const tasks = await provideNpmScriptsForFolder(context, path, showWarning);
+						yield path;
 						visitedPackageJsonFiles.add(path.fsPath);
-						allTasks.push(...tasks);
 					}
 				}
 			}
 		}
-		return allTasks;
 	} catch (error) {
```

Comment:
Not sure why it was added initially, but this catch is a no-op, you can remove the try/catch if you want